### ISSUE TITLE
domain of gcis:relatedProject is the intersection of publication and dataset

### ISSUE THREAD

**Issue by zednis** [USGCRP/gcis-ontology#143]
```
gcis:relatedProject a owl:ObjectProperty ;
    rdfs:label "related project" ;
    rdfs:comment "A publication or dataset may be the result of one or more projects." ;
    rdfs:domain gcis:Publication, gcis:Dataset ;
    rdfs:range gcis:Project ;
    rdfs:subPropertyOf prov:wasAttributedTo .
```

I believe this domain would be the intersection of `gcis:Publication` and `gcis:Dataset` rather than the union.

I think we should generalize the domain.

**Comment by justgo129** [comment]
Sounds logical to me but I'll defer to @rewolfe and @bduggan on this matter.

**Comment by zednis** [comment]
based on the discussion of in USGCRP/gcis/pull/235.

I suggest we change the domain to `prov:Entity` and update the comment on the property to be "A project the entity is attributed to."

```
gcis:relatedProject a owl:ObjectProperty ;
    rdfs:label "related project" ;
    rdfs:comment "A project the entity is attributed to." ;
    rdfs:domain prov:Entity;
    rdfs:range gcis:Project ;
    rdfs:subPropertyOf prov:wasAttributedTo .
```

**Comment by justgo129** [comment]
Fine here aside from the grammatical issue in the definition.

**Comment by justgo129** [comment]
Also, to what aspect of the Project is the entity attributed?  The existence, creation, etc?

**Comment by zednis** [comment]
Is there a difference between ascribing an entity's existence vs it's creation?

Definition of PROV Attribution:

> Attribution is the ascribing of an entity to an agent. When an entity e is attributed to agent ag, entity e was generated by some unspecified activity that in turn was associated to agent ag. Thus, this relation is useful when the activity is not known, or irrelevant.

Roles can be used to with a qualified Attribution (see example at http://www.w3.org/TR/prov-o/#Attribution) to specify the role the agent had in the activity that generated the entity.

**Comment by rewolfe** [comment]
Shouldn't it be "wasRelatedTo", not just "relatedTo"?

Change "A project the entity is attributed to."
to "A project that was related to an entity."

I assume the use cases are:

a. Project A was related to Project B.
b. Project A was related to Organization B.

However, should we be more specific and use terms like "wasAffiliatedWith",
"wasFundedBy" and "wasPartOf"?

Examples:

c. Project A was affiliated with Project B.
d. Project A was funded by Organization B.
e. Project A was part of Project B.
f. Project A was part of Organization B.

On Wed, Sep 2, 2015 at 11:22 AM, Stephan Zednik notifications@github.com
wrote:

> Is there a difference between ascribing an entity's existence vs it's
> creation?
> 
> Definition of PROV Attribution:
> 
> Attribution is the ascribing of an entity to an agent. When an entity e is
> attributed to agent ag, entity e was generated by some unspecified activity
> that in turn was associated to agent ag. Thus, this relation is useful when
> the activity is not known, or irrelevant.
> 
> Roles can be used to with a qualified Attribution (see example at
> http://www.w3.org/TR/prov-o/#Attribution) to specify the role the agent
> had in the activity that generated the entity.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/143#issuecomment-137128308
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by zednis** [comment]
This property is used to assign attribution of some entity to a project.  The super-property is `prov:wasAttributedTo` which has a specific meaning that holds between an entity and an agent and is more precise then a general 'related' property.

A general 'relatedTo' property would be a different property.

The two use cases @rewolfe provides are not appropriate a sub-property of `prov:wasAttributedTo`.  Additionally the previous domain of the property - which was constructed erroneously as an intersection - indicates that the domain of this property was intended to be types like `gcis:Publication` or `gcis:Dataset` and not projects.

**Comment by justgo129** [comment]
I'll defer to @rewolfe going forward with this ticket.

**Comment by rewolfe** [comment]
Okay, I am sorry that I misunderstood the context. So, an example would be:

  Dataset A was related to Project B.

I think this is pretty general, but it may be okay.  More typical term
would be "was produced by".

So to relate a dataset to a project, we need two triples:

  Dataset A was produced by Activity B.

  Activity B was related to Project C.

Again, a more typical term way of relating an activity and project would be:

  Activity B was performed under Project C.

or

  Activity B was part of Project C.

I'm not sure if this helps.

**Comment by zednis** [comment]
@rewolfe The current definition of `gcis:relatedProject` asserts that it is a sub-property of `prov:wasAttributedTo`.

This means that the statement
`:Dataset_A prov:wasAttributedTo :Project_C` 
is inferable from
`:Dataset_A gcis:relatedProject :Project_C`

In PROV attribution has the following definition - _Attribution is the ascribing of an entity to an agent. When an entity e is attributed to agent ag, entity e was generated by some unspecified activity that in turn was associated to agent ag. Thus, this relation is useful when the activity is not known, or irrelevant._

So in PROV attribution is a shorthand for saying:
`:Dataset_A prov:wasGeneratedBy :Activity_B`
`:Activity_B prov:wasAssociatedWith :Project_C`

This conforms to your suggested statements:

>   Dataset A was produced by Activity B.
>  Activity B was related to Project C.

**Comment by rewolfe** [comment]
@zednis Got it! Thanks for being patient in helping me understand this.

So, yes, I'm okay with the definition: "A project the entity is attributed
to."
However, would is be clearer?: "An entity ascribed to a project through an
unspecified activity."

On Fri, Sep 4, 2015 at 12:37 PM, Stephan Zednik notifications@github.com
wrote:

> @rewolfe https://github.com/rewolfe The current definition of
> gcis:relatedProject asserts that it is a sub-property of
> prov:wasAttributedTo.
> 
> This means that the statement
> :Dataset_A prov:wasAttributedTo :Project_C
> is inferable from
> :Dataset_A gcis:relatedProject :Project_C
> 
> In PROV attribution has the following definition - _Attribution is the
> ascribing of an entity to an agent. When an entity e is attributed to agent
> ag, entity e was generated by some unspecified activity that in turn was
> associated to agent ag. Thus, this relation is useful when the activity is
> not known, or irrelevant._
> 
> So in PROV attribution is a shorthand for saying:
> :Dataset_A prov:wasGeneratedBy :Activity_B
> :Activity_B prov:wasAssociatedWith :Project_C
> 
> This conforms to your suggested statements:
> 
> Dataset A was produced by Activity B.
> Activity B was related to Project C.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/143#issuecomment-137786205
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by zednis** [comment]
I am ok with "An entity ascribed to a project through an unspecified activity" if everyone else is.

I was trying to follow the grammatical structure that dublin core terms uses in their comments, but I don't think it works well in this case.

**Comment by justgo129** [comment]
:+1:

**Comment by rewolfe** [comment]
+1

On Fri, Sep 4, 2015 at 2:08 PM, justgo129 notifications@github.com wrote:

> [image: :+1:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/USGCRP/gcis-ontology/issues/143#issuecomment-137808218
> .

## 

Robert Wolfe, NASA GSFC @ USGCRP, o: 202-419-3470, m: 301-257-6966

**Comment by zednis** [comment]
ok, updated proposal with the new comment text:

```
gcis:relatedProject a owl:ObjectProperty ;
    rdfs:label "related project" ;
    rdfs:comment "An entity ascribed to a project through an unspecified activity" ;
    rdfs:domain prov:Entity;
    rdfs:range gcis:Project ;
    rdfs:subPropertyOf prov:wasAttributedTo .
```

Since I think we already have consensus I will wait a bit and baring any objections I will submit the pull request.

**Comment by xgmachina** [comment]
+1

**Comment by justgo129** [comment]
Closed #143 due to merged #147.

**Comment by justgo129** [comment]
@zednis @xgmachina now that #147 has been merged, please feel free to proceed editing the turtle templates where necessary.  It may also impact #17.
